src/metabase/db/spec.clj

```diff
@@ -14,6 +14,17 @@
           :subname db}
          (dissoc opts :db)))
 
+(defn csv
+  "Create a database specification for a csv database. Opts should include a key
+  for :db which is the path to the database file."
+  [{:keys [db]
+    :or {db "h2.db"}
```

Comment:
Why would your CSV file be named `h2.db?

Reply:
-> data.csv